common/src/wasm.rs

```diff
@@ -1,4 +1,4 @@
-use hyper::http::{HeaderMap, Method, Request, Response, StatusCode, Uri, Version};
+use hyper::http::{self, HeaderMap, Method, Request, Response, StatusCode, Uri, Version};
```

Comment:
`hyper::http` is actually just a re-export of the `http` crate. So we can make hyper a dev-dependency only